big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -77,9 +80,15 @@ init_per_group(GroupName, Config) ->
 end_per_group(GroupName, Config) ->
     metrics_helper:finalise_by_all_metrics_are_global(Config, GroupName =:= all_metrics_are_global).
 
+init_per_testcase(cluster_size = CN, Config) ->
+    Config1 = ensure_nodes_not_clustered(Config),
+    escalus:init_per_testcase(CN, Config1);
 init_per_testcase(CaseName, Config) ->
     escalus:init_per_testcase(CaseName, Config).
 
+end_per_testcase(cluster_size = CN, Config) ->
+    Config1 = ensure_nodes_clustered(Config),
```

Comment:
Do we need them clustered after the test case?

Reply:
This is as Kacper explained before:

> My intention here was to save the current state of MIM cluster, remove all nodes from the cluster for the sake of this particular test case and return to the same cluster state once the test case is finished.